### DISCUSSION TITLE
So i made an excel. Just before the update of this night. But i challenged the bot. So im logging

### DISCUSSION THREAD

**Discussion by BGB2k11** [kyleskom/NBA-Machine-Learning-Sports-Betting#165] [Show and tell]
![image](https://user-images.githubusercontent.com/47173321/228840579-5459074d-e1c1-4924-b40d-5e321afa8b99.png)

so its in dutch but the main takeaway:
-  Every bet is 5 euro. so red is -> no bet or lost bet. when the space in the odd screen is "white" i am not betting it. This means the AI and the XG model are contradicting itself.

ML - 50 euro bets on 10 matches -> so 5 euro a match. Revenue of 60euro -> profit of 10 euro
UO - 30 euro on 6 matches -> 56,7 euro win -> profit of 26,7 euro

When we take EV into account:

![image](https://user-images.githubusercontent.com/47173321/228841223-c7e73dc8-3968-4e82-af84-a7a421c8c783.png)

Again 5 euro lay in

only 1 bet cause the EV is not worth it 
ML - 1 bet 1 win -> 7.4euro reventue -> 2,4 euro profit
UO - 6 bets 6 wins -> 47,3 reventue -> 17,3 euro

if would have combined both bets -> so with EV and without EV combined
Lay in: 115 euro
Revenue: 171,45euro
Profit: 56,45 euro

This is a 49% folio gain. If your folio was 115 euro :) 
Not bad for a 1 nighter

**Comment by MaoZeLone** [comment]
This log is very interesting and helpful. Would you like to make similar records in the future？

**Reply by BGB2k11** [reply]
making one each day - except for the weekends since i can not log it in the weekends for now.

**Comment by BGB2k11** [comment]
Matchday 31/03 (this is 31/03 my time -> CET)

lost my Excel status. So its a printscreen :(

![image](https://user-images.githubusercontent.com/47173321/229060061-a619118b-602d-45a0-9d77-102d83e79e2b.png)

**Reply by MaoZeLone** [reply]
![image](https://user-images.githubusercontent.com/129410858/229066671-e1f9875b-280c-4f27-a51c-8ed63ce82f85.png)
This is my result, which is very different from yours. I am curious about the reason. Could you please give me your contact information and want to have a deeper communication with you

**Reply by szocpaul** [reply]
Hello, it's a very good tool to manage your bankroll instead of a simple excel sheet.
[https://app.bet-analytix.com/](url)

**Reply by BGB2k11** [reply]
@MaoZeLone 
I run  everything from scratch.
Getting stats
Retraining XGBoost ML & UO
Retraining AI models
Then run the script

You can connect to me on discord.BG#2466

@kyleskom and @nkgilley  is this the way to do it ?
Those steps every day ?

**Reply by BGB2k11** [reply]
> Hello, it's a very good tool to manage your bankroll instead of a simple excel sheet. [https://app.bet-analytix.com/](url)

Thanks !

https://app.bet-analytix.com/bankroll/936073

**Reply by nealmick** [reply]
I think retraining models everyday might not be the best idea.  You want a single model so that you can get a decent sample size of tests... Each time you retrain a model the weights are reset, and recalibrated over the training process... effectively making a new and unique model that will most likely get different results on the same games (at-least by a little).... Also the training data is not updated daily, covered in this discussion https://github.com/kyleskom/NBA-Machine-Learning-Sports-Betting/discussions/155

**Reply by BGB2k11** [reply]
> I think retraining models everyday might not be the best idea. You want a single model so that you can get a decent sample size of tests... Each time you retrain a model the weights are reset, and recalibrated over the training process... effectively making a new and unique model that will most likely get different results on the same games (at-least by a little).... Also the training data is not updated daily, covered in this discussion #155

So this is for the AI right ? The XG needs to be retrained every day right ?

**Reply by nealmick** [reply]
Nope, XG works the same way as far as I know, and would have the same issue if retrained daily….

**Reply by BGB2k11** [reply]
> Nope, XG works the same way as far as I know, and would have the same issue if retrained daily….

Thanks for the insight !
After how many days do we need to retrain then ?

**Reply by nealmick** [reply]
I don't think the NBA game changes much in the course of a few months, or even a few years...
